homeassistant/components/media_player/onkyo.py

```diff
@@ -71,9 +71,26 @@ def __init__(self, receiver, sources, name=None):
         self._reverse_mapping = {value: key for key, value in sources.items()}
         self.update()
 
+    def command(self, command):
+        """Run an eiscp command and catch connection errors."""
+        try:
+            result = self._receiver.command(command)
+        except (BlockingIOError, ValueError, OSError, ConnectionAbortedError,
```

Comment:
BlockingIOError and ConnectionAbortedError are sublasses of OSError. You can remove the first two.